torch/nn/utils/clip_grad.py

```diff
@@ -1,12 +1,12 @@
 
-def clip_grad_norm(parameters, max_norm, norm_type=2):
+def clip_grad_norm_(parameters, max_norm, norm_type=2):
```

Comment:
This needs a deprecating version. Something like below should work:
```
def clip_grad_norm(parameters, max_norm, norm_type=2):
    r"""Clips gradient norm of an iterable of parameters.
    
    .. warning:: This method is now deprecated in favor of :func:`torch.nn.utils.clip_grad_norm_`.
    """
    warnings.warn("nn.utils.clip_grad_norm is now deprecated in favor of nn.init.nn.utils.clip_grad_norm_.",
                  stacklevel=2)
    return clip_grad_norm_(parameters, max_norm, norm_type)
```

Reply:
Oh yes good point, thanks!